includes/class-wc-tracker.php

```diff
@@ -391,11 +435,80 @@ private static function get_all_template_overrides() {
 
 	/**
 	 * When an admin user logs in, there user agent is tracked in user meta and collected here.
+	 *
 	 * @return array
 	 */
 	private static function get_admin_user_agents() {
 		return array_filter( (array) get_option( 'woocommerce_tracker_ua', array() ) );
 	}
+
+	/**
+	 * Get order totals
+	 *
+	 * @return array
+	 */
+	private static function get_order_totals() {
+		global $wpdb;
+
+		$gross_total = $wpdb->get_var( "
+			SELECT
+				SUM( order_meta.meta_value ) AS 'gross_total'
+			FROM {$wpdb->prefix}posts AS orders
+			LEFT JOIN {$wpdb->prefix}postmeta AS order_meta ON order_meta.post_id = orders.ID
+			WHERE order_meta.meta_key =  '_order_total'
+				AND orders.post_status =  'wc-completed'
+			GROUP BY order_meta.meta_key
+		" );
+
+		if ( is_null( $gross_total ) ) {
+			$gross_total = 0;
+		}
+
+		return array(
+			'gross' => $gross_total,
+		);
+	}
+
+	/**
+	 * Get last order date
+	 *
+	 * @return string
+	 */
+	private static function get_last_order_date() {
```

Comment:
Why not do this with direct SQL like `get_order_totals`? Since it's internal we can - it will be faster as you're only querying the posts table.

Reply:
That would also allow you to get both dates (first/last) at the same time using min/max.